addons/stock/wizard/stock_picking_return.py

```diff
@@ -113,6 +113,7 @@ def _prepare_move_default_values(self, return_line, new_picking):
 
     def _prepare_picking_default_values(self):
         vals = {
+            'partner_id': self.picking_id.partner_id.id,
```

Comment:
Not sure about this one. The partner_id on return picking have implications in subcontracting. What about updateing the _compute_location_id in case of return. Or force the location after the compute is triggered ?